docs/glossary-and-resources.md

```diff
@@ -23,23 +23,44 @@
 - **interchange format** - syntax/file format used to convert the inputs of message formatting into inputs for other systems (ex: l10n systems).
 - **interpolate** - inserting the contents of one string in the middle of another, at places indicated by a pattern / placeholder. See _translation merging_.
+- **LDML** - Locale Data Markup Language - the spec schema that defines CLDR
 - **locale fallback** - offering a reasonable substitute locale when the requested locale's resources are not available. Results may vary depending on context (ex: audio vs. text vs. video).
 - **locale matching** - computing the locale fallback.
+- **markup** - characters present in the MessageFormat source strings that are intended to provide structure, functionality, or formatting to the output string, as opposed to display text.
+- **match** - a mechanism in MessageFormat that allows the message author to specify a variety of outcomes based on the combined values of one or more inputs.
+- **metadata** - Addition information about an MF2 message that aids in translation, or other pre-runtime processes, but has no impact on how the message will be rendered. This may be context notes, screenshots of the string in situ, etc. Metadata should never be relied upon at runtime.
+- **open/close** - syntax within the source test that changes the execution context of the parser from output mode to code mode, and back again.
+- **operand** - A character in a message that denotes an action taken during runtime execution, such as value assignment or comparison.
+- **options** - values passed in, along with an MF2 message, that will be used to determine the output. Examples may be item count in a search results string.
+- **placeholder** - Syntax in the MF2 message that deontes that an option should be inserted into that place in the string.
+- **registry** - MF2's function registry, which allows reusable logic to be executed at runtime while resolving an MF2 string to output.
 - **resource** - files bundled with an application that are loaded in by the executable code. UI strings, etc. and their locale-specific translations are typically stored as resources.
 - **roundtrip** - the process of transforming a message into another format or representation, then transforming it back into the original format.
 - **translation merging** - in l10n TMSes, the document-level interpolation of translated content. In other words, the replacing of translatable units in the source document with their equivalent translated units. See _interpolate_.
 - **selector** - see _placeholder type_.
 - **specification** - the rules we decide that describe what is passed to the API for message formatting (structure of data, syntax, etc.).
 - **serialization** - how to convert in-memory representations of data to/from a file/stream.
+- **sigil** - A character in a string that is used to denote a change in context.
```

Comment:
This isn't quite right; they used to be the leading prefix character, but now with standalone markup we also use one in a trailing position.